casper/src/test/rholang/FailingResultCollectorTest.rho

```diff
@@ -5,16 +5,16 @@ new assert(`rho:test:assertAck`),
     ch1, ch2,
     ack1, ack2, ack3, ack4, ack5, ack6
 in {
-  assert!("false should fail", false, "clue assertTrue", *ack1) |
-  assert!("0 != 1 should fail", (0, "==", 1), "clue assertEquals", *ack2) |
-  assert!("incorrect assertion should fail", ("some", "garbage", "assertion", "content"), "bad assertion should fail", *ack3) |
+  assert!("false should fail", 1, false, "clue assertTrue", *ack1) |
```

Comment:
It was confusing to me to see `assert` behave this way and I had to go to `FailingResultCollectorSpec.scala` to see why. A comment that the normal `RhoSpec` behavior is not used here would probably help.

Reply:
I added a comment at the top of the file to make this clear.